network/shuffle/pom.xml

```diff
@@ -43,6 +43,17 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
+      <artifactId>leveldbjni-all</artifactId>
+      <version>1.8</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
I think it's better to add it, especially since you're explicitly using jackson annotations in the code. (It should follow the usual pattern - declare in root pom's dependencyManagement, use wherever needed).